packages/dashboard-frontend/src/services/bootstrap/index.ts

```diff
@@ -84,6 +85,9 @@ export default class Bootstrap {
       this.fetchClusterInfo(),
       this.fetchClusterConfig(),
     ]);
+
+    this.checkWorkspaceStopped();
```

Comment:
I think the best place for ```checkWorkspaceStopped``` will be next:

```
...
      this.updateDevWorkspaceTemplates(settings),
      this.fetchWorkspaces().then(() => this.checkWorkspaceStopped()),
      this.fetchClusterInfo(),
...
```

Reply:
WDYT?

Reply:
+-1

Reply:
it would not
+1 to @olexii4 proposal